bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/profiles/RawRockerToOnOffProfile.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2014,2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.smarthome.core.thing.internal.profiles;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.CommonTriggerEvents;
+import org.eclipse.smarthome.core.thing.profiles.ProfileCallback;
+import org.eclipse.smarthome.core.thing.profiles.ProfileTypeUID;
+import org.eclipse.smarthome.core.thing.profiles.SystemProfiles;
+import org.eclipse.smarthome.core.thing.profiles.TriggerProfile;
+import org.eclipse.smarthome.core.types.State;
+
+/**
+ * The {@link RawRockerToOnOffProfile} transforms rocker switch channel events into ON and OFF commands.
+ *
+ * @author Jan Kemmler - Initial contribution
+ */
+public class RawRockerToOnOffProfile implements TriggerProfile {
+
+    private final ProfileCallback callback;
+
+    RawRockerToOnOffProfile(ProfileCallback callback) {
+        this.callback = callback;
+    }
+
+    @Override
+    public ProfileTypeUID getProfileTypeUID() {
+        return SystemProfiles.RAWROCKER_TO_ON_OFF;
+    }
+
+    /**
```

Comment:
please don't repeat the javadoc of an overridden method (unless the meaning somehow changed, of course). It's already hard to keep it up-to-date at all, and I fear it's gonna be a complete mess with copies like this.